### ISSUE TITLE
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Non-zero status code returned while running Concat node. Name:'Concat_113' Status Message: concat.cc:154 PrepareForCompute Non concat axis dimensions must match: Axis 2 has mismatched dimensions of 33 and 34

### ISSUE THREAD

**Issue by seawater668** [onnx/onnx#5134]
I have successfully converted the onnx of dynamic input and output. If the width and height of the input image are the same, no error will be reported. When the width and height of the input are inconsistent, the above error will be reported. What is the reason?

**Comment by jcwchen** [comment]
Hi @seawater668,
Did you use dynamic_axes when exporting ONNX model from PyTorch? Could you please provide the model and the code to repro this issue?

**Comment by seawater668** [comment]
@jcwchen 
I use dynamic_axes when exporting ONNX model from PyTorch,
 model_name = model_path.split("/")[-1]
    model_dir = os.path.dirname(model_path)
    print(model_dir, model_name)

    onnx_name = model_name.split('.')[0] + "_dy.onnx"
    onnx_file = os.path.join(model_dir, onnx_name)

    my_model = load_learner(model_dir, file=model_name)
    fastai_model = my_model.model.eval()  # gets the PyTorch model
    batch_size = 1  # 随机的取值，当设置dynamic_axes后影响不大
    dummy_input = torch.randn(batch_size, 3, 512, 512).cuda()
    # dummy_input = torch.randn(batch_size, 3, 512, 512, requires_grad=True).cuda()
    fastai_output= fastai_model(dummy_input)

    input_names = ['input']
    output_names = ['output']
    dynamic_axes = {input_names[0]: {0: 'batch_size', 2 : 'in_width', 3: 'int_height'},
                    output_names[0]: {0: 'batch_size', 2: 'out_width', 3:'out_height'}
                    }
    remove_all_spectral_norm(fastai_model)  # function posted above by 'jantic'
    torch.onnx.export(
        fastai_model,
        dummy_input,
        onnx_file,
        do_constant_folding=True,
        export_params=True,
        input_names=input_names,
        output_names=output_names,
        dynamic_axes=dynamic_axes,
        opset_version=13
    )

**Comment by justinchuby** [comment]
If the issue persists, please raise it in the PyTorch repository. Thanks!
